rich/pretty.py

```diff
@@ -503,6 +503,25 @@ def iter_rich_args(rich_args: Any) -> Iterable[Union[Any, Tuple[str, Any]]]:
                 else:
                     yield arg
 
+        original_getattr_method = None
```

Comment:
Not keen on this solution. It is probably better to respect that an object is doing magic, but be more robust in handling errors.

Reply:
I saw your patch, that's way better ! Thx !